data/deep missions.txt

```diff
@@ -0,0 +1,1354 @@
+		or
+			"deep convoy" >= 2
+			has "Deep: Remnant Research: done"
+	on offer
+		conversation
+			``
+
+
+
+mission "Deep: Remnant 0"
+	minor
+	name `Fetching Keystones`
+	description `Buy 10 Quantum Keystones from Hai space and return them to <destination>, where Ivan and his team will research them and their link to the spatial anomaly in Terminus.`
+	source
+		attributes deep
+		near "Epsilon Leonis" 1 100
+	to offer
+		"deep convoy" >= 1
+		has "Deep: Mystery Cubes: done"
+		has "Terminus exploration: done"
+		or
+			"deep convoy" == 3
+			has "Deep: TMBR 2: done"
+	on offer
+		conversation
+			`While wandering through the spaceport, you are approached by a man you instantly recognize as the scientist whose drone you recovered data from in Terminus, thanks to his comically thick glasses. He introduces himself as Ivan and mentions he heard how helpful you have been to others within the Deep.`
+			`	"I was hoping you would continue assisting us in our research on the spatial anomaly in Terminus. Thanks to the data you retrieved, we have deduced that the anomaly is in fact some sort of wormhole, but it is far too unstable to support sending anything through it."`
+			`	"I must ask you, do you know of the Hai?" You nod. "Splendid! Then you will be able to help. One of the scientists on my team grew up in their territory, and suggested that the so called "Quantum Keystones" which they sell might have something to do with the wormhole. He recalls hearing some Hai folktales of these stones unlocking previously impassible pathways, soothing the minds and bodies of those who possessed them. Would you kindly retrieve ten of them for us, so that we may be able to discover if these legends are true, and they allow access into the wormhole?"`
+			branch remnant
+				has "First Contact: Remnant: offered"
+			choice
+				`	"I would be glad to help."`
+				`	"Sorry, but Hai space is too far for me."`
+					decline
+					
+			`	"Thank you again, Captain!" he says very enthusiastically. "I'd have traveled myself, but I must confess we are low on funds and will not receive more unless we make some sort of discovery. We will be able to pay for half the cost of the stones once you return."`
+				accept
+			
+			label remnant
+			choice
+				`	"I already know what is on the other side of the wormhole. In fact, installing a single Quantum Keystone is enough to enter into the spatial anomaly."`
+				`	"Sorry, but Hai space is too far for me right now."`
+					decline
+			apply
+				set "knew of stones before scientists"
+			`	Ivan looks like he is about to fall over with joy after you mention having entered the wormhole. "Really? Please, tell me what you found!"`
+			branch friendly
+				has "event: ember waste label"
+			`	You tell Ivan all that you saw on the other side of the wormhole, and how you found strange-looking humans who communicate in song and sign language, seemed obsessed with blood, and attacked you after you wouldn't let them have some of yours.`
```

Comment:
I phrased it to be a bit of a comical hot-take. This player literally had to run away from some possible aliens who wanted to prod him with a strange device that collects blood. That they were then willing to kill him because he decided to not let strange devices near his body *is* a little obsessive 😆